### PR TITLE
Update 05-Loix10.md

### PR DESCRIPTION
**Pull request by Loix10**
Added new PR

### CODE REVIEW

**Review by pacmxias — APPROVED** [review]
its ok

**Review by pacmxias — APPROVED** [review]

**Review by Loix10 — COMMENTED** [review]
Added emoji

**Review by pacmxias — APPROVED** [review]

**Comment by Loix10** [comment]
Slide merge

File: _slides/05-Loix10.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hi I am Luis Leal, Muito gosto em estar Convosco!!👍
```
Comment (pacmxias) [inline _slides/05-Loix10.md:7]: ok

**Pull request merged**